gamestonk_terminal/prediction_techniques/README-gpu-accel.md

```diff
@@ -0,0 +1,50 @@
+# GPU acceleration with CUDA
+
+Most up-to-date docs and recommendations: https://www.tensorflow.org/install/gpu
+
+### CUDA 11.0.3, cuDNN 8.0.5 - Tested working
+
+1. Install [CUDA 11.0.3](https://developer.nvidia.com/cuda-toolkit-archive)
+
+2. Install [cuDNN 8.0.5 for CUDA 11.0](https://developer.nvidia.com/rdp/cudnn-archive). This requires signing up for the nvidia developer program. To install, copy the bin/, lib/, include/ dirs into your CUDA install dirs.
+
+    | Platform | CUDA path                                                  |
+    | -------- | ---------------------------------------------------------- |
+    | Linux    | /usr/local/cuda-X.Y                                        |
+    | Windows  | C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v<XX.Y> |
+
+3. Set the environment variables `CUDA_PATH` and `CUDA_PATH_V11` if needed. These are usually set by the CUDA installer.
+
+### CUDA 11.1, cuDNN 8.0.5 - not tested
+
+### CUDA 11.2.1, cuDNN 8.1 - does not work
```

Comment:
Haha -- try [this](https://github.com/tensorflow/tensorflow/issues/44777#issuecomment-771285431). Tried on CUDA 11.2.

Reply:
I actually tried this but I was still running into out-of-memory issues fairly often despite having far more VRAM than needed for the models I was using. May have been due to cuDNN 8.1, may have been easy to resolve, I didn't fight it for long.

Reply:
Hmmm... Here is my info for the system:

I used [this](https://github.com/archlinux/svntogit-community/commit/84a21cead1005d1f11c908b7c136fdf1f7c3ad0a#diff-3e341d2d9c67be01819b25b25d5e53ea3cdf3a38d28846cda85a195eb9b7203a) version to install on an `archlinux` system. The command is `sudo pacman -S cuda cudnn python-tensorflow-opt-cuda`.

The output of `nvidia-smi` gives:
```
Tue Mar 16 19:33:17 2021       
+-----------------------------------------------------------------------------+
| NVIDIA-SMI 460.56       Driver Version: 460.56       CUDA Version: 11.2     |
```

The output of `nvcc --version` gives:
```
nvcc: NVIDIA (R) Cuda compiler driver
Copyright (c) 2005-2021 NVIDIA Corporation
Built on Thu_Jan_28_19:32:09_PST_2021
Cuda compilation tools, release 11.2, V11.2.142
Build cuda_11.2.r11.2/compiler.29558016_0
```

And finally the `cat` of `/usr/include/cudnn_version.h` gives
```c++
#define CUDNN_MAJOR 8
#define CUDNN_MINOR 1
#define CUDNN_PATCHLEVEL 0
```

So I *think* CUDA 11.2.1, and cuDNN 8.1 is tested and proved to be working.